tests/table/TableTests.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using Bunit;
+using Xunit;
+
+namespace AntDesign.Tests.Table
+{
+    public class TableTests : AntDesignTestBase
+    {
+        class Person
+        {
+<<<<<<< Updated upstream
```

Comment:
Is there some conflict that hasn't been resolved?

Reply:
I think I did something weird last night when trying to push. 😂 It's fixed now.